mppm/install.md

```diff
@@ -4,10 +4,11 @@ title: Install Multiplayer Play Mode
 description: How to install Multiplayer Play Mode
 ---
 
-To begin testing your multiplayer functionality, you must first install the Multiplayer Play Mode package.
+You can use Multiplayer Play Mode in Unity Editor versions 2023.1 or later.
```

Comment:
You specified 1.1.0 earlier; I like this more generic version better, but should they be consistent?

Reply:
I see what you mean. I worded it like this because version 1.0.0  is generically compatible, however, version 1.1.0 is _only_ compatible with that specific Unity version. So I actually need to update this line to match the other one.